gwak/train/dataloader.py

```diff
@@ -573,10 +573,10 @@ def generate_waveforms(self, batch_size, parameters=None, ras=None, decs=None):
                     dec=decs[i] if decs is not None else None
                 )
             if signal_class == "CCSN":
-                self.generate_waveforms_ccsn(
+                responses, dec, phic = self.generate_waveforms_ccsn(
```

Comment:
@AndyC80297 see here. Pls check if params and ra can also be provided by CCSN generation